e2e/testdata/fn-eval/out-of-place-fnchain-stdout/.expected/exec.sh

```diff
@@ -0,0 +1,7 @@
+#! /bin/bash
+
+set -eo pipefail
+
+kpt fn eval --image gcr.io/kpt-fn/set-namespace:v0.1.3 -o stdout -- namespace=staging \
```

Comment:
Make sure the book is updated in this PR: https://kpt.dev/book/04-using-functions/02-imperative-function-execution?id=chaining-functions-using-the-unix-pipe

And do a fast release so docs match impl.

Reply:
Okay.